tests/instruments/test_cors.py

```diff
@@ -78,7 +78,7 @@ def test_fastapi_cors_bootstrap() -> None:
     assert len(fastapi_application.user_middleware) == 1
     assert isinstance(fastapi_application.user_middleware[0], Middleware)
     cors_middleware: typing.Final = fastapi_application.user_middleware[0]
-    assert cors_middleware.cls is CORSMiddleware
+    assert cors_middleware.cls is CORSMiddleware  # type: ignore[comparison-overlap]
```

Comment:
Typing issue may disappear if we use `isinstance`:

```suggestion
    assert isinstance(cors_middleware.cls, CORSMiddleware)
```

Reply:
getting Argument 1 to "issubclass" has incompatible type "_MiddlewareFactory[P]"; expected "type"  [arg-type]

Reply:
isinstance failing, issubclass working, but raises

Argument 1 to "issubclass" has incompatible type "_MiddlewareFactory[P]"; expected "type"  [arg-type]

Reply:
seems not very important